src/gui/spatialArrow.js

```diff
@@ -0,0 +1,324 @@
+createNameSpace("realityEditor.gui.spatialArrow");
+
+import * as THREE from '../../thirdPartyCode/three/three.module.js';
+
+(function (exports) {
+    
+    let canvasContainer;
+    let canvas;
+    let ctx;
+    let screenW, screenH;
+    let screenRatio;
+    
+    let color, colorLighter;
+
+    function initService() {
+        console.log('%c spatial arrow init!', 'color: green');
+        addCanvas();
+        resizeCanvas();
+        initCanvas();
+        update();
+    }
+
+    // for testing purpose on js fiddle, don't include it in final code
+    const windowRatio = 1;
```

Comment:
should this be removed?

Reply:
Yes. I used another variable screenBorderFactor to calculate a small offset for the arrows to be drawn inside the screen. Forgot to remove this one.